test/queryParser_test.js

```diff
@@ -333,6 +333,13 @@ describe('queryParser', () => {
       expect(results['left']['term_max']).to.equal('2017-11-18T04:28:11.999Z');
       expect(results['left']['inclusive']).to.equal('right');
     });
+
+    it('parses a quoted term in a range expression', () => {
```

Comment:
Can you please also add a `toString()` test case? Thank you!

Reply:
I don't have bandwidth at the moment.

@ddio Since you're interested perhaps you'd like to add a `toString()` test case?